src/java.desktop/share/classes/javax/swing/table/DefaultTableColumnModel.java

```diff
@@ -290,6 +290,8 @@ public int getColumnIndex(Object identifier) {
      * @param   columnIndex     the index of the column desired
      * @return  the <code>TableColumn</code> object for the column
      *                          at <code>columnIndex</code>
+     * @exception  ArrayIndexOutOfBoundsException if <code>columnIndex</code>
+     *             is not in valid range (< 0 or >= getColumnCount())
```

Comment:
```suggestion
     * @exception  ArrayIndexOutOfBoundsException if <code>columnIndex</code>
     *             is not in the valid range (< 0 or >= getColumnCount())
```
I find it confusing: the expression in parenthesis seems to clarify the meaning of *the valid range* but doesn't.

I think it's better to make it clearer:
```suggestion
     * @exception  ArrayIndexOutOfBoundsException if <code>columnIndex</code>
     *             is out of range (<code>columnIndex < 0 || columnIndex >= getColumnCount()</code>)
```